extensions/one-time-password/src/one-time-password.tsx

```diff
@@ -159,18 +159,18 @@ export default () => {
   function getCopyToClipboardContent(secret: string) {
     try {
       return generateToken(secret);
-    } catch (err) {
+    } catch {
       return '';
     }
   }
 
   const globalActions = (
     <ActionPanel.Section>
       <ActionPanel.Submenu title="Create New" icon={Icon.Plus} shortcut={{ modifiers: ['cmd'], key: 'n' }}>
-        <Action title="Scan a QR Code" icon={Icon.Camera} onAction={() => scanQRCode('scan')} />
+        <Action title="Scan a Qr Code" icon={Icon.Camera} onAction={() => scanQRCode('scan')} />
```

Comment:
Can we revert these changes please?

Reply:
lint erros :| 
<img width="986" height="100" alt="{D42A745B-31B4-40B3-A80A-D155F8A9C3B6}" src="https://github.com/user-attachments/assets/98ba32ea-c086-44cd-b8da-7e3d0561b38c" />